apps/logger/logger_118.go

```diff
@@ -0,0 +1,20 @@
+//go:build go1.18 && !go1.20
+
+package logger
+
+import "fmt"
+
+// Logger struct for Go versions <= 1.20.
+type Logger struct {
+	logCallback CallbackFunc
```

Comment:
I was under the impression that Azure SDK will not be using the callback for now so there is no need for it. When they upgrade to 1.20, they will start using slog